src/Integration.UnitTests/MefServices/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -96,7 +95,7 @@ public void ActiveSolutionBoundTracker_Initialisation_Unbound()
             host.VisualStateManager.ClearBoundProject();
 
             // Act
-            using (var testSubject = CreateTestSubject(this.host, this.activeSolutionTracker, loggerMock.Object))
+            using (var testSubject = CreateTestSubject(this.host, this.activeSolutionTracker, this.configProvider, loggerMock.Object))
```

Comment:
The setup for this test is ugly, but I took the easy way out and did the simplest fixup possible due to time constraints.